### ISSUE TITLE
How is the performance of the baseline?

### ISSUE THREAD

**Issue by Ailln** [Axe--/Visual-Question-Answering#1]
1. train loss
2. train accurate
3. validate accurate

**Comment by Axe--** [comment]
Hey, please see the results for the following configuration:

python3 main.py
--mode train
--model baseline
--expt_dir /home/axe/Projects/VQA_baseline/results_log
--expt_name K_1000
--run_name O1_wrk_8_bs_512
--train_img /home/axe/Datasets/VQA_Dataset/raw/train2014
--train_file /home/axe/Datasets/VQA_Dataset/processed/vqa_train2014.txt
--val_img /home/axe/Datasets/VQA_Dataset/raw/val2014
--val_file /home/axe/Datasets/VQA_Dataset/processed/vqa_val2014.txt
--num_cls 1000
--vocab_file /home/axe/Datasets/VQA_Dataset/processed/vocab_count_5_K_1000.pickle
--batch_size 512
--num_epochs 50
--learning_rate 0.0001
--log_interval 100
--save_interval 1000
--val_size 10000
--K_eval 1000
--model_ckpt None
--vgg_wts_path /home/axe/Projects/Pre_Trained_Models/vgg11_bn-6002323d.pth
--vgg_train False
--gpu_id 0
--opt_lvl 1
--num_workers 8

Validation Accuracy: 52.34 %  || Validation Loss: 2.0480
Epoch [25/50], Step [20800/866], Loss: 1.4018 | time elapsed: 9.52h | time left: 10.29h

_Note: The model was stopped while it was converging (training incomplete). Also replacing VGG-11 with VGG-19 would lead to improved accuracy._

**Comment by Ailln** [comment]
thanks!

**Comment by Ailln** [comment]
by the way, how is the performance of the HieCoAttn model?

**Comment by Axe--** [comment]
Validation Accuracy: 53.43 %  || Validation Loss: 1.5999
Epoch [9/50], Step [22200/2773], Loss: 1.1730 | time elapsed: 9.29h | time left: 48.75h
(The reported accuracy: ~60%)

The model wasn't converging further, hence stopped early. 
Also I didn't had the time to perform any hyperparameter tuning/debugging for reproducibility, due to some emergency.

Will resume this implementation in a couple of weeks.

**Comment by Ailln** [comment]
thanks for your reply, I have benefited a lot from your work. 

have a good day!
